compair/static/modules/course/course-module.js

```diff
@@ -463,11 +454,9 @@ module.controller(
                 Session.expirePermissions();
 
                 var course = ret;
-                if ($scope.closeDuplicate) {
-                    $scope.closeDuplicate(course.id);
-                } else {
-                    $scope.$close(course.id);
-                }
+                submitted = false;
+                $location.path('/course/' + ret.id);
```

Comment:
Sorry, I missed this the first time but this breaks LTI links. the course id needs to be passed back to the LTI controller via modal closing in order for the new duplicated course to be linked to the lti context

Reply:
Ok, thanks. Added that in now.